Core1Platform/02_CoreCapabilities.md

```diff
@@ -0,0 +1,51 @@
+# Core capabilities
+
+The *Actindo Core1 Platform* provides you with the following:
+- [Standalone solution or modernization of legacy ERP systems](md#standalone-solution-or-modernization-of-legacy-erp-systems)
+- [Omni-channel integration](#omni-channel-orchestration)
+- [Data transformation and process orchestration](#data-transformation-and-process-orchestration)
+- [Monitoring and reporting](#monitoring-and-reporting)
+ 
+
+## Standalone solution or modernization of legacy ERP systems
```

Comment:
What do you think about calling it "Use Cases" or "Fields of application"

Reply:
done